frontend/src/components/organizations/NotForProfitQuestions.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+  <v-skeleton-loader :loading="loading" type="table-tbody">
+    <v-card class="my-4 pa-4 w-100" variant="outlined">
+      <v-container fluid class="pa-0">
+        <AppLabel>Not for Profit Information</AppLabel>
+        <v-card variant="outlined" class="card-outline px-2 py-6 my-4">
+          <v-row no-gutters>
+            <v-col>
+              <AppLabel>Date of Incorporation</AppLabel>
+            </v-col>
+          </v-row>
+          <AppDateInput id="date-to" v-model="model.dateOfIncorporation" :rules="[...rules.required, rules.MMDDYYYY]" :max="todayDate" :disabled="false" label="Select The Date" class="mt-6 mx-6" />
+        </v-card>
+        <v-card v-if="isReferenceLetterRequired" variant="outlined" class="card-outline px-2 py-6 my-4 w-100">
+          <v-row no-gutters>
+            <v-col cols="7" md="3" lg="2">
+              <AppLabel>Upload Documents</AppLabel>
+            </v-col>
+            <v-col>
+              <v-tooltip
+                content-class="tooltip"
+                text="A letter from a community representative or agency that has knowledge about the sector and can demonstrate how the society provides services to support the local community."
+                max-width="300px">
+                <template #activator="{ props }">
+                  <v-icon size="large" v-bind="props">mdi-information-slab-circle-outline</v-icon>
+                </template>
+              </v-tooltip>
+            </v-col>
+          </v-row>
+          <v-card variant="outlined" class="card-outline px-2 py-6 my-4 w-100">
+            <AppDocumentUpload
+              v-model="model.documentsToUpload"
+              :uploaded-documents="model.uploadedDocuments"
+              :document-type="DOCUMENT_TYPES.COMMUNITY_LETTER"
+              :document-label="'Letter of Reference for Community Support'"
+              entity-name="accounts"
+              :disabled="false"
+              :loading="loading"
+              :rules="[...rules.required]"
+              @delete-uploaded-document="deleteUploadedDocument"></AppDocumentUpload>
+            <div v-if="!areDocumentsComplete">
```

Comment:
we can use AppMissingInfoError here. Also, we should update the text to be similar to other error messages in application (e.g.: Document upload of your Community Support Letter required)
` <AppMissingInfoError v-if="!areDocumentsComplete">{{  APPLICATION_ERROR_MESSAGES.COMMUNITY_SUPPORT_LETTER }}</AppMissingInfoError>`

Reply:
done